backends/ebpf/psa/ebpfPsaTable.cpp

```diff
@@ -23,12 +23,36 @@ limitations under the License.
 
 namespace EBPF {
 
+template<class F>
+class EBPFTablePsaPropertyVisitor : public Inspector {
+ protected:
+    F functor;  // called for every property entry
```

Comment:
seems to be called for their values which are PathExpression only.

Reply:
What do you mean? The comment should be clarified or something else?

Reply:
Actually the right way to do this is to have this be a base class and define a new subclass for each different analysis, with just a preorder(PathExpression*) method. Please rewrite it that way.